haeinsa-core/src/main/java/kr/co/vcnc/haeinsa/HaeinsaTransaction.java

```diff
@@ -0,0 +1,716 @@
+/**
+ * Copyright (C) 2013-2015 VCNC Inc.
+ *
+ * <p>Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file
+ * except in compliance with the License. You may obtain a copy of the License at
+ *
+ * <p>http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * <p>Unless required by applicable law or agreed to in writing, software distributed under the
+ * License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package kr.co.vcnc.haeinsa;
+
+import com.google.common.base.Preconditions;
+import com.google.common.collect.ComparisonChain;
+import com.google.common.collect.Maps;
+import com.google.common.hash.HashFunction;
+import com.google.common.hash.Hashing;
+import java.io.IOException;
+import java.util.*;
+import java.util.Map.Entry;
+import java.util.concurrent.atomic.AtomicBoolean;
+import kr.co.vcnc.haeinsa.HaeinsaTransactionLocal.HaeinsaTransactionLocals;
+import kr.co.vcnc.haeinsa.exception.ConflictException;
+import kr.co.vcnc.haeinsa.exception.RecoverableConflictException;
+import kr.co.vcnc.haeinsa.thrift.generated.TRowKey;
+import kr.co.vcnc.haeinsa.thrift.generated.TRowLock;
+import kr.co.vcnc.haeinsa.thrift.generated.TRowLockState;
+import org.apache.hadoop.hbase.util.Bytes;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * ***********************START NEUTRONIC ADDITION***************** 1) Added getMutationRows()
+ * method 2) Added getId() method ************************END NEUTRONIC ADDITION*****************
+ */
```

Comment:
Just a nit. Spotless mangled this in a few files. Not sure if it's worth fixing but it is harder to read now. It was:
```
/*************************START NEUTRONIC ADDITION*****************
 * 1) Added getMutationRows() method
 * 2) Added getId() method
 *************************END NEUTRONIC ADDITION******************/
```